HealthcareApp/src/Healthcare.Application/Appointments/Commands/Book/BookAppointmentCommandHandler.cs

```diff
@@ -37,6 +37,7 @@ public async Task<Result<string>> Handle(BookAppointmentCommand request, Cancell
         
         var appointment = _mapper.Map<Appointment>(request);
         appointment.EndTime = request.StartTime.Add(TimeSpan.FromMinutes(15));
+        appointment.DurationMinutes = (int)(appointment.EndTime - appointment.StartTime).TotalMinutes;
```

Comment:
You can avoid explicit type casting here

Reply:
Unfortunately tried, but without casting it does not works